### PR TITLE
completed task

### PR DESCRIPTION
**Pull request by Andriy-Shymkiv**
- [DEMO LINK](https://andriy-shymkiv.github.io/layout_catalog/)
- [TEST REPORT LINK](https://andriy-shymkiv.github.io/layout_catalog/report/html_report/)

@mykola-koval  the same problem as i have with product card

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,12 +1,388 @@
...
+    <header class="header">
+
+      <div>
+        <a href="#home">
+          <img src="./images/logo.png" alt="moyo">
+        </a>
+      </div>
+
+      <nav class="nav">
+
```
Comment (Artem66) [inline src/index.html:21]: remove space

File: src/index.html [hunk 2]
```diff
@@ -1,12 +1,388 @@
...
+          <li class="nav__item">
+            <a href="#photo" class="nav__link">Photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#video" class="nav__link">Video</a>
+          </li>
+        </ul>
+
```
Comment (Artem66) [inline src/index.html:61]: remove space

File: src/index.html [hunk 3]
```diff
@@ -1,12 +1,388 @@
...
+            <div class="stars__star stars__star--active"></div>
+            <div class="stars__star stars__star--active"></div>
+            <div class="stars__star stars__star--active"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="card__review">Reviews: 5</div>
+        </div>
+
+        <div class="card__price">
+          <span class="card__price-title">Price</span>
```
Comment (Artem66) [inline src/index.html:252]: You have to use another tag for text `p`

File: src/index.html [hunk 4]
```diff
@@ -1,12 +1,388 @@
...
+        </a>
+
+      </div>
+
+      <div class="card">
+
+        <img
+          src="./images/imac.jpeg"
+          alt="imac"
+          class="card__logo"
```
Comment (Artem66) [inline src/index.html:111]: It would be better another name
```suggestion
          class="card__image"
```

**Review by Artem66 — APPROVED** [review]
Good job! I recommend you to change some issues

File: src/index.html [hunk 5]
```diff
@@ -1,12 +1,388 @@
...
+
+        <div class="card__rating">
+          <div class="stars">
+            <div class="stars__star stars__star--active"></div>
+            <div class="stars__star stars__star--active"></div>
+            <div class="stars__star stars__star--active"></div>
+            <div class="stars__star stars__star--active"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="card__review">Reviews: 5</div>
```
Comment (andreyolin) [inline src/index.html:128]: </div>

           <div class="card__review">Reviews: 5</div>

need space here i think

File: src/index.html [hunk 6]
```diff
@@ -1,12 +1,388 @@
...
```
Comment (andreyolin) [inline src/index.html:168]: </div>

       <div class="card__review">Reviews: 5</div>
need space here i think

File: src/index.html [hunk 7]
```diff
@@ -1,12 +1,388 @@
...
```
Comment (andreyolin) [inline src/index.html:208]: </div>

           <div class="card__review">Reviews: 5</div>

need space here i think.
Also you can use p here

File: src/index.html [hunk 8]
```diff
@@ -1,12 +1,388 @@
...
```
Comment (andreyolin) [inline src/index.html:248]: </div>

           <div class="card__review">Reviews: 5</div>

need space here i think.
Also you can use p here

File: src/index.html [hunk 9]
```diff
@@ -1,12 +1,388 @@
...
```
Comment (andreyolin) [inline src/index.html:288]: </div>

           <div class="card__review">Reviews: 5</div>

need space here i think.
Also you can use p here

File: src/index.html [hunk 10]
```diff
@@ -1,12 +1,388 @@
...
```
Comment (andreyolin) [inline src/index.html:328]: </div>

           <div class="card__review">Reviews: 5</div>

need space here i think.
Also you can use p here

**Pull request closed without merging**